libs/icu_headroom_metric.py

```diff
@@ -20,6 +20,12 @@
 NJ_CORRECTION = (
     0.176 - 0.57 + DEFAULT_ICU_DECOMP
 )  # https://trello.com/c/T15w5VLq/418-nj-icu-occupancy-rate-nonsensical
```

Comment:
Very small nit, but when black reformats trailing comments on a single line to be a parens wrapped line, I prefer to put the comment above on its own line

Reply:
Changed to have it with the rest of the comment below. Seems fine with me.

At the same time, I'm going to give you a little bit of a hard time for choosing black as our linter ("Black is the uncompromising Python code formatter. By using it, you agree to cede control over minutiae of hand-formatting") but then still having formatting opinions :) .